.github/workflows/main.yml

```diff
@@ -0,0 +1,79 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI Build
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  
+  linux:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+    - uses: actions/checkout@v2
+    - name: wget
+      uses: wei/wget@v1
+      with:
+        args: -O nuget.exe https://dist.nuget.org/win-x86-commandline/latest/nuget.exe
+    
+    - name: Install Dependencies
+      run: |
+        mono nuget.exe install
+        mono nuget.exe restore
```

Comment:
I think that only the restore is needed, can you try removing the `mono nuget.exe install` line?